wip-0020.md

```diff
@@ -130,11 +130,14 @@ A reference implementation for the proposed protocol improvement can be found in
 
 ## Adoption Plan
 
-An adoption plan will be proposed upon moving this document from the _Draft_ stage to the _Proposed_ stage.
+An activation date is proposed for April 5 2022 at 9am UTC, that is, protocol epoch #1032960.
+
+From the perspective of [TAPI], this proposal will be signaled along with the WIP-0021. This proposal adheres to the base implicit signal, and its first signaling epoch is also set to that of WIP-0021.
```

Comment:
There's a contradiction here. It is either signaled along WIP-0021 (bit 2) or adheres to the base implicit signal (bit 0). You can simply copy and paste the same paragraph from WIP-0021 here.